code/datums/wires/vending.dm

```diff
@@ -46,6 +42,7 @@
 		if(WIRE_CONTRABAND)
 			V.categories ^= CAT_HIDDEN
 		if(WIRE_SHOCK)
+			V.shock(usr, 50)
```

Comment:
`usr` isn't acceptable, this proc receives the `user` already as a parameter, adjust the proc signature and use that instead